beets/library.py

```diff
@@ -259,7 +259,7 @@ class LibModel(dbcore.Model):
     """Shared concrete functionality for Items and Albums.
     """
 
-    _format_config_key = None
+    format_config_key = None
```

Comment:
I changed it to avoid setting a underscore-prefixed property (a.k.a "protected") from outside. If you think that'd be preferable I'll change it back.